scripts/auto-rebase/rebase.py

```diff
@@ -350,6 +432,10 @@ def get_token(org, repo):
 
 
 def main():
```

Comment:
At Line#447, the suggestion is to change 
```python
REMOTE_DRY_RUN = False if try_get_env(DRY_RUN_ENV, die=False) == "" else True
```
line due to `R1719: The if expression can be replaced with 'not test' (simplifiable-if-expression)` issue.

I think this line can be changed to 

```python
REMOTE_DRY_RUN = bool(try_get_env(DRY_RUN_ENV, die=False))
```

is there any better approach to re-write this ?

Reply:
Yeah, that's also better. Addressed.